tests/airdrop.ts

```diff
@@ -23,14 +25,19 @@ describe('airdrop', () => {
   const merkleVerifierProgram = anchor.workspace.MerkleVerifier as Program<MerkleVerifier>;
   const governanceVerifierProgram = new Program(governance_verifier_idl as Idl, new PublicKey('ATCsJvzSbHaJj3a9uKTRHSoD8ZmWPfeC3sYxzcJJHTM5'), provider);
   const amount = new anchor.BN(1_000_000);
+  const verifierSeed = crypto.randomBytes(32);
```

Comment:
How is this different from publickey? What's the benefit of removing publickey below too?